trompace/mutations/controlaction.py

```diff
@@ -137,3 +137,14 @@ def mutation_request_controlaction(controlaction_id: str, entrypoint_id: str, pr
         }
     }
     return format_mutation("RequestControlAction", args)
+
+def mutation_update_controlaction(controlaction_id: str, actionStatus: trompace.StringConstant):
```

Comment:
I would name this a bit more specifically because it's used just to update the status. How about `mutation_update_controlaction_status`? 

For the status parameter, in python we typically use _ instead of upper-case characters, so `action_status`. 
For the type, we have some enums in the CE: https://github.com/trompamusic/trompa-ce-client/blob/c7111c601c23aeace322816aa5d306a4ee1a1f69/trompace/constants.py#L9
it's be great to accept this enum, and then in the code use `"actionStatus": StringConstant(str(action_status))` to convert the enum name into a constant for the mutation. (I think this is right, you might need to change it)

Can you add some documentation here?